cli/pkg/local/app.go

```diff
@@ -78,20 +78,20 @@ func NewApp(ctx context.Context, ver config.Version, verbose bool, olapDriver, o
 	}
 
 	// Create a local runtime with an in-memory metastore
-	globalDrivers := []*runtime.Connector{
+	systemConnectors := []*runtimev1.Connector{
 		{
-			Type:    "sqlite",
-			Name:    "metastore",
-			Configs: map[string]string{"dsn": "file:rill?mode=memory&cache=shared"},
+			Type:   "sqlite",
+			Name:   "metastore",
+			Config: map[string]string{"dsn": "file:rill?mode=memory&cache=shared"},
 		},
 	}
 
 	rtOpts := &runtime.Options{
 		ConnectionCacheSize: 100,
-		MetastoreDriver:     "metastore",
+		MetastoreDriver:     "sqlite",
 		QueryCacheSizeBytes: int64(datasize.MB * 100),
 		AllowHostAccess:     true,
-		GlobalDrivers:       globalDrivers,
+		SystemConnectors:    systemConnectors,
```

Comment:
The name is "metastore", but the `MetastoreDriver` is "sqlite" – does this work?